src/field_theory/minpoly/gcd_monoid.lean

```diff
@@ -24,27 +24,37 @@ This file specializes the theory of minpoly to the case of an algebra over a GCD
  * `gcd_domain_unique` : The minimal polynomial of an element `x` is uniquely characterized by
     its defining property: if there is another monic polynomial of minimal degree that has `x` as a
     root, then this polynomial is equal to the minimal polynomial of `x`.
+
+ * `is_integrally_closed_eq_field_fractions`: For integrally closed domains, the minimal polynomial
+    over the ring is the same as the minimal polynomial over the fraction field.
+
+## Todo
+
+ * Remove all results that have been generalized from `normalized_gcd_monoid` to
+    `is_integrally_closed`.
```

Comment:
This sentence looks weird, you mean removing the results that are now special cases of something more general?

Reply:
Yep that was the idea, I'll try and rephrase it!